src/test/regress/sql/multi_create_table.sql

```diff
@@ -266,3 +266,15 @@ SELECT shardcount FROM pg_dist_colocation WHERE colocationid IN
 );
 
 DROP TABLE shard_count_table_3;
+
+-- test column compression propagation in table distribution
+CREATE TABLE col_compression (a TEXT COMPRESSION pglz, b TEXT);
```

Comment:
can you make sure that online/offline rebalancer also creates compression after moving shards?